src/services/PromotionService.ts

```diff
@@ -25,21 +25,17 @@ import GooglePlacesService from './GooglePlacesService';
  *
```

Comment:
We can remove the unused import above!

Reply:
Ohya good point! We'll need to add it back once we merge in the BE sort PR